### DISCUSSION TITLE
Lost Historical Traffic Stats data

### DISCUSSION THREAD

**Discussion by IMXEren** [zhongyang219/TrafficMonitor#1494] [Q&A]
My Traffic Monitor program files were located in C: Drive and "Configuration and data files" was set "Save to the program directory". Suddenly after a pc restart, it was changed to "Save to Appdata directory", so I changed the option to program directory and clicked apply. Further, I copied (replaced) all these files -
1. config.ini
2. error.log
3. history_traffic.dat
4. history_traffic.dat.bak
    to the traffic monitor program directory. Then, after the program restart I lost all data of "Historical Traffic Statistics". I recovered the all the original program directory files of traffic monitor and want to ask is there any way I can restore them or recover my data.

Recovered ![recovered](https://user-images.githubusercontent.com/96839938/209472178-276309b0-445e-4f0d-84c1-ffce37888918.png)

Current ![current](https://user-images.githubusercontent.com/96839938/209472179-4d7a1e55-e64e-46a9-b5ba-df24df5ad3d6.png)

**Comment by QiangShang** [comment]
use notepad ,open history_traffic.dat.
Then copy the old history data which you don`t have to your current history_traffic.dat file

**Reply by IMXEren** [reply]
Thanks for replying.
I did exactly what you said but it didn't change the history stats data and I'll just let it slide.
